test/docs/testplans/vnet_to_vnet.md

```diff
@@ -0,0 +1,60 @@
+- Depending on the test cases, a combination of UDP traffic for IPv4, IPv6 addresses will be generated
+
+### Test Cases
+[TODO: Update Table]
+
+| Test Description                                             | Expected Result               |
+| :----------------------------------------------------------- | :------------------------ |
+| UDP IPv4 inbound  | Verify traffic originating from one VNet simulated terminates on another simulated VNet. <br><br> Verify counters in DUT in the inbound direction are accurately incremented. <br><br> Verify latency measurements <br> Verify packet loss is 0 (Tx frames - Rx frames = 0) <br><br> Verify configured throughput rate is achieved in traffic statistics  |
+| UDP IPv4 outbound | Verify traffic originating from one VNet simulated terminates on another simulated VNet. <br><br> Verify counters in DUT in the inbound direction are accurately incremented. <br><br> Verify latency measurements <br> Verify packet loss is 0 (Tx frames - Rx frames = 0) <br><br> Verify configured throughput rate is achieved in traffic statistics |
+| UDP IPv4 bidirectional  | Verify traffic originating from one VNet simulated terminates on another simulated VNet. <br><br> Verify counters in DUT in the inbound direction are accurately incremented. <br><br> Verify latency measurements <br> Verify packet loss is 0 (Tx frames - Rx frames = 0) <br><br> Verify configured throughput rate is achieved in traffic statistics    |
+| UDP IPv6 inbound | Verify traffic originating from one VNet simulated terminates on another simulated VNet. <br><br> Verify counters in DUT in the inbound direction are accurately incremented. <br><br> Verify latency measurements <br> Verify packet loss is 0 (Tx frames - Rx frames = 0) <br><br> Verify configured throughput rate is achieved in traffic statistics    |
+| UDP IPv6 outbound direction | Verify traffic originating from one VNet simulated terminates on another simulated VNet. <br><br> Verify counters in DUT in the inbound direction are accurately incremented. <br><br> Verify latency measurements <br> Verify packet loss is 0 (Tx frames - Rx frames = 0) <br><br> Verify configured throughput rate is achieved in traffic statistics  |
+| UDP IPv6 bidirectional  | Verify traffic originating from one VNet simulated terminates on another simulated VNet. <br><br> Verify counters in DUT in the inbound direction are accurately incremented. <br><br> Verify latency measurements <br> Verify packet loss is 0 (Tx frames - Rx frames = 0) <br><br> Verify configured throughput rate is achieved in traffic statistics  |                                                 
+|  ACL inbound | Verify data traffic is allowed and unwanted traffic based on ACL is dropped. <br><br>Verify ACL inbound counters on DUT are incremented
```

Comment:
Please specify whether IPv4 or IPv6 or both; if both, suggest you have table row for each permutation [IPv4, IPv6] x [Inbound, Outbound] i.e. 4 test cases.

Reply:
Each IPv4 and IPv6 inbound/outbound is mentioned separately. As we add more testcases to the table all the permutation options will be mentioned.